### ISSUE TITLE
Login redirect on address pi.hole non working after upgrading to Version v4.3.5 (Error 400 Bad Request)

### ISSUE THREAD

**Issue by stefanostatuti** [pi-hole/pi-hole#3165]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**
5
`{Replace this with a number from 1 to 10. 1 being not familiar, and 10 being very familiar}`

---
**Expected behaviour:**
when a user try to navigate to pi.hole (or using the IPAddress of device) should be redirected to http://pi.hole/admin/

**Actual behaviour:**
the redirect doesn't works and the user receive error 400 Bad Request
![400BadRequest](https://user-images.githubusercontent.com/4499372/75271836-23f7b480-57fd-11ea-982b-df39b2b8ea9f.PNG)

**Steps to reproduce:**
navigate on pi.hole address

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

https://tricorder.pi-hole.net/pkgk1dbo20

**Troubleshooting undertaken, and/or other relevant information:**

`{Steps of what you have done to fix this}`

**Comment by dschaper** [comment]
The `lighttpd` configuration files have not been changed. This is what I see from your debug.

```conf
-rw-r--r-- 1 root root 3499 feb 24 22:30 /etc/lighttpd/lighttpd.conf
   server.modules = (
   	"mod_access",
   	"mod_accesslog",
   	"mod_auth",
   	"mod_expire",
   	"mod_compress",
   	"mod_redirect",
   	"mod_setenv",
   	"mod_rewrite"
   )
   server.document-root        = "/var/www/html"
   server.error-handler-404    = "pihole/custom.php"
   server.upload-dirs          = ( "/var/cache/lighttpd/uploads" )
   server.errorlog             = "/var/log/lighttpd/error.log"
   server.pid-file             = "/var/run/lighttpd.pid"
   server.username             = "www-data"
   server.groupname            = "www-data"
   server.port                 = 80
   accesslog.filename          = "/var/log/lighttpd/access.log"
   accesslog.format            = "%{%s}t|%V|%r|%s|%b"
   index-file.names            = ( "index.php", "index.html", "index.lighttpd.html" )
   url.access-deny             = ( "~", ".inc", ".md", ".yml", ".ini" )
   static-file.exclude-extensions = ( ".php", ".pl", ".fcgi" )
   compress.cache-dir          = "/var/cache/lighttpd/compress/"
   compress.filetype           = ( "application/javascript", "text/css", "text/html", "text/plain" )
   mimetype.assign   = ( ".png"  => "image/png",
                         ".jpg"  => "image/jpeg",
                         ".jpeg" => "image/jpeg",
                         ".html" => "text/html",
                         ".css" => "text/css; charset=utf-8",
                         ".js" => "application/javascript",
                         ".json" => "application/json",
                         ".txt"  => "text/plain",
                         ".svg"  => "image/svg+xml" )
   include_shell "/usr/share/lighttpd/use-ipv6.pl " + server.port
   include_shell "find /etc/lighttpd/conf-enabled -name '*.conf' -a ! -name 'letsencrypt.conf' -printf 'include \"%p\"
' 2>/dev/null"
   $HTTP["url"] =~ "^/admin/" {
       
       setenv.add-response-header = (
           "X-Pi-hole" => "The Pi-hole Web interface is working!",
           "X-Frame-Options" => "DENY"
       )
       $HTTP["url"] =~ ".ttf$" {
           
           setenv.add-response-header = ( "Access-Control-Allow-Origin" => "*" )
       }
   }
   $HTTP["url"] =~ "^/admin/\.(.*)" {
        url.access-deny = ("")
   }
   include_shell "cat external.conf 2>/dev/null"
```

This is the section that redirects:

```config
   $HTTP["url"] =~ "^/admin/" {
       
       setenv.add-response-header = (
           "X-Pi-hole" => "The Pi-hole Web interface is working!",
           "X-Frame-Options" => "DENY"
       )
```

**Comment by dschaper** [comment]
Oh, here is the issue:

```log
-rw-r--r-- 1 www-data www-data 4061 feb 25 18:34 /var/log/lighttpd/error.log
   2020-02-24 22:45:26: (response.c.404) uri-path does not begin with '/': pihole/custom.php -> 400 
   2020-02-24 22:45:26: (response.c.404) uri-path does not begin with '/': pihole/custom.php -> 400 
   2020-02-24 22:47:25: (response.c.404) uri-path does not begin with '/': pihole/custom.php -> 400 
   2020-02-24 22:47:25: (response.c.404) uri-path does not begin with '/': pihole/custom.php -> 400 
   2020-02-24 22:47:40: (response.c.404) uri-path does not begin with '/': pihole/custom.php -> 400 
```
It looks like your `/etc/lighttpd/lighttp.conf` file is missing a fix from v4.3.2

https://github.com/pi-hole/pi-hole/pull/2826/files

**Comment by stefanostatuti** [comment]
hi, i've modified the /etc/lighttpd/lighttp.conf adding the missing / and now it's works as expected. I can close the ticket.
thanks.
